v3ElectronicsUserGuide.md

```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
     - Note this will kill any running process on the hardware, but if you're updating FW no one should be using the system anyway.
  31.  Summarize the actions you took in the elog entry you have already opened.
 
+## Preparing For a Power Cut
+
+In order to safely prepare a GEM test stand before a planned power cut execute:
+
+ 1. Power down the High Voltage.
+     - Use the opportunity to power down other sensitive equipement such as PMT's.
+ 2. Power down the Low Voltage.
+ 3. Place the µTCA modules in extraction mode:
+     - Gently pull the hot swap tab on the AMC's (CTP7 and AMC13). Wait until the blue LED stays on solid on each AMC's.
+     - Gently pull the hot swap tab on the MCH. Again, wait until the blue LED stays solid on.
+ 5. Power down the µTCA crate Power Modules one at a time. If powered with AC/DC converters, simply flip the switch on the AC/DC one at a time.
```

Comment:
All crates will be powered with an AC/DC converter.  The converter supplies the voltage to the PM.  The difference is some manufacturers provide crates with built in converters (e.g. VadaTech) while others require an external converter (e.g. Schroff).  Try to clarify this point.